repository/dimensions_repository.go

```diff
@@ -0,0 +1,132 @@
+package repository
```

Comment:
All the arguments to methods in this package are from the model package - suggests to me that they should be one package together?